platforms/ide/ide-native/src/integTest/groovy/org/gradle/ide/visualstudio/VisualStudioIncrementalIntegrationTest.groovy

```diff
@@ -189,8 +184,8 @@ class VisualStudioIncrementalIntegrationTest extends AbstractVisualStudioIntegra
         run "visualStudio"
 
         then:
-        skipped ":appVisualStudioSolution"
-        executedAndNotSkipped getComponentTasks("app")
+        skipped ":appVisualStudioSolution", getFiltersTask("app")
```

Comment:
Changed the tasks to more precisely declare their inputs, and the filters file is now skipped because the change does not affect its output.